### DISCUSSION TITLE
我想请问一下，生成ISO之后能不能用meta的面板替换自带的面板

### DISCUSSION THREAD

**Discussion by JJFJJML** [kkkgo/PaoPaoGateWay#83] [问和答]
好像自带的面板不怎么好看。。。
还有一个就是这个Gateway能不能使用proxy-providers的方式来设置节点
谢谢大佬

**Comment by kkkgo** [comment]
1、不能，面板是功能定制和性能优化的，你喜欢可以自己搭个面板自己填API。
2、具体功能跟ppgw无关，取决于你的核心支不支持。
